source/sessions/driver-sessions.rst

```diff
@@ -213,8 +213,10 @@ A ``ClientSession`` MUST be associated with a ``ServerSession`` at the time
 to the rule that a server session MUST NOT be used by two ``ClientSession``
 instances at the same time (see the Server Session Pool section).
 
-``startSession`` MUST report an error if sessions are not supported by the
-deployment (see How to Check Whether a Deployment Supports Sessions).
+Drivers MUST report an error if sessions are not supported by the deployment
```

Comment:
I'm OK with mentioning this under `startSession`, as it still covers half of the possible implementation and there is no better place short of creating an entirely new section just for this paragraph.

Reply:
I'm not seeing any text in the spec at all about other reasons sessions might not be supported.
related to that potential error case though, the spec does says drivers may call the server's `startSession` command to generate LSIDs if they cannot be generated locally.

Reply:
This may be an implementation detail for libmongoc, then. I don't believe it has any fallback to executing the `startSession` command and just requires the driver to be built with crypto. In any event, likely not relevant to this PR.